src/test/java/no/nav/veilarboppfolging/service/SisteEndringPaaOppfolgingBrukerServiceTest.java

```diff
@@ -0,0 +1,56 @@
+package no.nav.veilarboppfolging.service;
+
+import no.nav.common.types.identer.Fnr;
+import no.nav.veilarboppfolging.repository.SisteEndringPaaOppfolgingBrukerRepository;
+import no.nav.veilarboppfolging.test.IsolatedDatabaseTest;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+
+public class SisteEndringPaaOppfolgingBrukerServiceTest extends IsolatedDatabaseTest {
+    private SisteEndringPaaOppfolgingBrukerRepository sisteEndringPaaOppfolgingBrukerRepository;
+    private SisteEndringPaaOppfolgingBrukerService sisteEndringPaaOppfolgingBrukerService;
+
+    @Before
+    public void setUp() {
+        sisteEndringPaaOppfolgingBrukerRepository = new SisteEndringPaaOppfolgingBrukerRepository(db);
+        sisteEndringPaaOppfolgingBrukerService = new SisteEndringPaaOppfolgingBrukerService(sisteEndringPaaOppfolgingBrukerRepository);
+    }
+
+    @Test
+    public void testLagreSisteEndring() {
```

Comment:
Kanskje en test til som lagrer flere endringer for samme fnr, slik at den logikken også blir testet?

Reply:
Yep, jeg er enig..fikser..